lib/ui5Framework/Sapui5MavenSnapshotResolver.js

```diff
@@ -136,6 +136,7 @@ class Sapui5MavenSnapshotResolver extends AbstractResolver {
 		snapshotEndpointUrl = process.env.UI5_MAVEN_SNAPSHOT_ENDPOINT_URL || snapshotEndpointUrl;
 
 		if (!snapshotEndpointUrl) {
+			// Return an unexecuted promise instead of the resolved URL here.
```

Comment:
I would describe it as a function that returns a Promise which resolves with the URL.

Reply:
Done